tests/unit/test_storage.py

```diff
@@ -201,10 +281,23 @@ def test_ceph_base_interfaces(self):
 
 class TestStoragePluginPartCephDaemonChecks(StorageTestsBase):
 
-    def test_get_crushmap_mixed_buckets(self):
-        inst = ceph_daemon_checks.CephOSDChecks()
-        inst()
-        self.assertFalse('mixed_crush_buckets' in inst.output['ceph'])
+    @mock.patch.object(ceph_daemon_checks, 'issue_utils')
+    def test_get_crushmap_mixed_buckets(self, mock_issue_utils):
+        with mock.patch.object(ceph_core, 'CLIHelper') as mock_helper:
+            mock_helper.return_value = mock.MagicMock()
+            mock_helper.return_value.ceph_osd_crush_dump_json_decoded.\
+                return_value = json.loads(CEPH_OSD_CRUSH_DUMP)
+            inst = ceph_daemon_checks.CephOSDChecks()
+            inst.get_crushmap_mixed_buckets()
+            self.assertTrue(mock_issue_utils.add_issue.called)
+
+    @mock.patch.object(ceph_daemon_checks, 'issue_utils')
+    def test_get_crushmap_no_mixed_buckets(self, mock_issue_utils):
+        with mock.patch.object(ceph_core, 'CLIHelper') as mock_helper:
+            mock_helper.return_value = mock.MagicMock()
+            inst = ceph_daemon_checks.CephOSDChecks()
```

Comment:
since the existing data does not have mixed buckets there is no need to mock cli_helpers so you can remove the above 3 lines.

Reply:
ack, done